apps/microtvm/reference-vm/zephyr/Vagrantfile

```diff
@@ -16,7 +16,8 @@
 # under the License.
 
 Vagrant.configure("2") do |config|
-  config.vm.box = "tlcpack/microtvm-zephyr-2.7"
+  config.vm.box = "tlcpack/microtvm-zephyr"
+  config.vm.box_version = "0.5.0"
```

Comment:
what's the 0.5.0 signify? should we adopt 2.7.x as the version number?

Reply:
0.5.0 was just an incremental number. I like your suggestion

Reply:
so I guess 2.7 is the zephyr version and .X is just an incremental number?

Reply:
yeah then at least we retain the ability to specify the zephyr version in the box version scheme

Reply:
I'm not a huge fan of the 2.7.X convention. For one, I don't think it's clear where the Zephyr version part ends and the incremental number begins, and it would be easy to believe e.g. 2.7.1 was a full version number.

Additionally, the Arduino VM won't have a relevant version number, so its `box_version` is bound to be purely incremental. For consistency, we might want to make the Zephyr `box_version` purely incremental as well.

Reply:
I think 2.7.X as version number provides some information which is useful. X would be totally incremental because out RVM updates are not aligned with the minor version of Zephyr. 
We could also use arduino CLI version or SDK for arduino RVM.

Reply:
For Arduino, using the CLI version doesn't make much sense, as it does not affect how code is compiled. We can't use the SDK version either, as we have one SDK for each board we support. My preference is still to have the version number be purely incremental, but I'll defer to you.

Reply:
I agree with that. For a box version like X.Y.Z I think we can keep X.Y aligned with Zephyr version, but loose the Z because there could be cases where we don't update the Zephyr minor version but we want to release boxes for other updates. So we can keep Z totally incremental based on our needs.

Reply:
I agree that we want to support semantic versioning. This pretty much guarantees that the Arduino RVM version has to be purely incremental, because there are so many version numbers that matter. 

For Zephyr, I'd still prefer a purely incremental version number for consistency, but anything that supports semantic versioning seems reasonable too.

Reply:
we had discussion about RVM versioning. The conclusion was to be compatible semantic versioning we need to go fully incremental staring with version `1.0.0`. The reason is that there are cases where we want to do a major update on RVM without changing the Zephyr SDK version, in this case semantic versioning requires to release a new major version however if we want to keep Zephyr SDK version we cannot be backward compatible.